### PR TITLE
WiimoteEmu: Implement "Drawsome" tablet.

### PR DESCRIPTION
**Pull request by jordan-woyak**
We might as well emulate this too..

### CODE REVIEW

**Comment by JMC47** [comment]
Works perfectly in the *three*(*) drawsome titles I have.

*There are two drawsome titles and then a compilation of those two titles.

File: Source/Core/Core/CMakeLists.txt [hunk 1]
```diff
@@ -144,6 +144,7 @@ add_library(core
   HW/WiimoteEmu/MotionPlus.cpp
   HW/WiimoteEmu/Speaker.cpp
   HW/WiimoteEmu/Extension/Classic.cpp
+  HW/WiimoteEmu/Extension/DrawsomeTablet.cpp
```
Comment (spycrab) [inline Source/Core/Core/CMakeLists.txt]: Please add the header file to the sources as well

Comment (jordan-woyak) [reply Source/Core/Core/CMakeLists.txt]: Right there in the same list? The other headers aren't here (yet?)?

File: Source/Core/Core/HW/WiimoteEmu/Extension/DrawsomeTablet.h [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    // Pen X/Y is little endian.
+    u8 stylus_x1;
+    u8 stylus_x2;
+
+    u8 stylus_y1;
+    u8 stylus_y2;
+
+    u8 pressure1;
+
+    u8 pressure2 : 3;
```
Comment (leoetlino) [inline Source/Core/Core/HW/WiimoteEmu/Extension/DrawsomeTablet.h]: IIRC, new code should prefer BitField over C bitfields.

Comment (jordan-woyak) [reply Source/Core/Core/HW/WiimoteEmu/Extension/DrawsomeTablet.h]: Fixed.

**Pull request merged**